### ISSUE TITLE
InlineQueryResultArticle changed in new version

### ISSUE THREAD

**Issue by leocabeza** [atipugin/telegram-bot-ruby#47]
On the previous version you could do:

`Telegram::Bot::Types::InlineQueryResultArticle
  .new(id: 1, title: 'First article', message_text: 'Very interesting text goes here.')`

now the new version requires a InputMessageContent object:

```
input_message_content = Telegram::Bot::Types::InputTextMessageContent.new(message_text: 'Very interesting text goes here'
)

Telegram::Bot::Types::InlineQueryResultArticle.new(id: 1, title: 'First article', input_message_content: input_message_content)
```

However, when I do that... I get the following error:

> Telegram API has returned the error. (ok: "false", error_code: "400", description: "[Error]: Bad request: Field "input_message_content" must be of type Object")

Am I doing anything wrong? And also, the example in [Inline Bots](https://github.com/atipugin/telegram-bot-ruby#inline-bots) is outdated.

**Comment by atipugin** [comment]
Yes, it's definitely a bug. Going to fix it asap.

**Comment by atipugin** [comment]
Released 0.5.0.beta3 with fix. Readme is also updated. Check it out.

**Comment by leocabeza** [comment]
It's fixed now! Thanks.
